### PR TITLE
Update to bitorrent v4.3.8 and nordvpn issues fixed

### PR DESCRIPTION
**Pull request by ezruneko**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,14 +23,16 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
 COPY rootfs /
 
 # hadolint ignore=DL3018
-RUN addgroup -S openvpn \
+RUN apk update \
```
Comment (guillaumedsde) [inline Dockerfile:26]: this should not be needed since we use `apk add --no-cache` later on

File: rootfs/usr/sbin/install_qbittorrent.sh [hunk 2]
```diff
@@ -3,8 +3,8 @@ set -x
 ARCH="$(uname -m)"
 echo "building for ${ARCH}"
 
-QBITTORRENT_VERSION=4.3.8
-LIBTORRENT_VERSION=v2.0.4
+QBITTORRENT_VERSION=4.3.9
+LIBTORRENT_VERSION=v1.2.14
```
Comment (guillaumedsde) [inline rootfs/usr/sbin/install_qbittorrent.sh:7]: I've taken care of this in another PR

**Comment by guillaumedsde** [comment]
Hi, sorry for the delayed reply, I've commented on some minor points, running the script with bash should solve the issue indeed, nicely spotted

**Comment by ezruneko** [comment]
done!

**Comment by guillaumedsde** [comment]
apologies again for the delay, I'll look this over after the holidays

**Comment by guillaumedsde** [comment]
Thanks again for the contribution, I've merged these changes into the development branch, I'll merge them into master in a week or so (in the meantime the changes will be available in the development tagged image)

**Comment by JonDror** [comment]
@guillaumedsde can this be merged into master? can't use NordVPN without it

**Comment by pranoy10** [comment]
> @guillaumedsde can this be merged into master? can't use NordVPN without it

Have you found a temporary workaround to get this resolved?

**Comment by MorGuux** [comment]
> Thanks again for the contribution, I've merged these changes into the development branch, I'll merge them into master in a week or so (in the meantime the changes will be available in the development tagged image)

Any update on this? I can use the development branch fine, but since the python branch is separate, I don't have a functional search bar.

**Comment by KaiDevrim** [comment]
Update?

**Comment by guillaumedsde** [comment]
Good evening, sorry I've not been very responsive on this, the change has been merged into master, I'll make sure the python based image is also built for the development branch in the future

I've struggled with my git rebase so github doesn't recognize this PR as merged (the rebase modified some commits due to conflicts) so I'm closing it because it is merged (see commits 17c9540d24cd71b584c50381daaaf5fd7d3e05f5 )

thanks again for the contribution, I'll make sure to rebase correctly next time so github shows this PR as merged instead of closed

**Pull request closed without merging**